packages/serve/package.json

```diff
@@ -23,6 +23,14 @@
     "@types/node": "12.7.2",
     "typescript": "3.5.3"
   },
+  "peerDependencies": {
+    "webpack-dev-server": ">=3.0.0"
+  },
+  "peerDependenciesMeta": {
+    "webpack-dev-server": {
+      "optional": false
+    }
+  },
```

Comment:
I think we can simplify this and just remove, because something in `peerDependencies` always mean non optional :smile:

Reply:
I guess we can make adjustments in another PR.